src/Components/NavBar.js

```diff
@@ -0,0 +1,15 @@
+import { NavLink } from 'react-router-dom';
+import "./NavBar.css";
+
+const NavBar = () => {
+    return(
+        <nav className="NavBarContainer">
+            <NavLink to={"/"}><img alt="" src="https://i.pinimg.com/236x/37/b6/16/37b616503c897d4fdf4315a319b72e74--agora-sunflowers.jpg"/></NavLink>
```

Comment:
Add an exact to the / NavLink so the active class isn't triggered on all routes.